apps/src/templates/ProjectTemplateWorkspaceIconV2.tsx

```diff
@@ -55,8 +56,10 @@ const ProjectTemplateWorkspaceIconV2: React.FunctionComponent<
       }}
       tooltipOverlayClassName={className}
     >
-      {/* FontAwesomeV6Icon does not work with WithTooltip. */}
-      <i className={'fa-kit fa-connected-level'} />
+      {/* FontAwesomeV6Icon with custom icon does not work with WithTooltip here. */}
```

Comment:
`FontAwesomeV6Icon is used in `WithTooltip` here: https://github.com/code-dot-org/code-dot-org/blob/4ba8b5eaa1f1795b781b5b8f2c0a28c1e8b2d598/apps/src/aichat/views/InfoTooltipIcon.tsx#L34
But the custom icon for some reason is not working. cc: @levadadenys

Reply:
We've posted almost the same time 😄

Reply:
Yeah, it's strange that it works in one place and is not working in other place.
Though, it's also tricky in my opinion to wrap `<i>` with `<button>` element when <button> is not clickable.

Can you please try pulling the latest staging and smth like:
```
<WithTooltip ... >
  <FontAwesomeV6Icon ... />
 </ WithTooltip>
```

If that won't work - let's discuss it in DM, but I think it will work

Reply:
Pulled in staging and fixed! Props were not assigned correctly on my end.